napari/utils/colormaps/colormap_utils.py

```diff
@@ -360,21 +362,20 @@ def increment_unnamed_colormap(
     return name
 
 
-def ensure_colormap_tuple(colormap: ValidColormapArg) -> Tuple[str, Colormap]:
+def ensure_colormap(colormap: ValidColormapArg) -> Colormap:
     """Accept any valid colormap arg, and return (name, Colormap), or raise.
 
     Adds any new colormaps to AVAILABLE_COLORMAPS in the process.
 
     Parameters
     ----------
     colormap : ValidColormapArg
-        colormap as str, Colormap, {name: Colormap} ``dict``, or
-        (name, Colormap) ``tuple``.
+        colormap as str, napari.utils.Colormap.
```

Comment:
I have added, but had to move type definition into colormaps folder to avoid circular imports. This is probably not a long term ideal solution but seems like a reasonable step for now